solr/contrib/ltr/src/java/org/apache/solr/ltr/ranking/ModelQuery.java

```diff
@@ -240,9 +239,12 @@ public Explanation explain(LeafReaderContext context, int doc)
         explanations[index++] = feature.explain(context, doc);
       }
 
+      final List<Normalizer> norms = meta.getNorms();
```

Comment:
Should the ModelWeight be fetching the norms here?  It doesn't for scoring, so why does it need it for explaining (explain and score should pretty much be the exact same).  Should the LTRScoringAlgo handle the explain part for norm?